### PR TITLE
Image component

### PR DESCRIPTION
**Pull request by malaykhakhar**

### CODE REVIEW

File: components/Image/image.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        </div>
+
+        <h1>Round Image</h1>
+        <br>
+        <br>
+        <img src="/components/images/responsive-image.jpg" alt="responsive-image" class="img-round">
+
+        <iframe
+          src="https://carbon.now.sh/embed/6sofNhu8BVaNrTKYnusb"
+          style="width: 680px; height: 353px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (Rishabh-078) [inline components/Image/image.html:58]: It was mentioned in LMS that we must avoid using inline styling.
So this sizing can be shifted to the stylesheet.

**Review by Rishabh-078 — CHANGES_REQUESTED** [review]
Rest looks fine to me.
Just the inline styling thing that I have commented on.

File: components/Image/image.html (hunk 1 above)
Comment (malaykhakhar) [reply components/Image/image.html:58]: Hey so this is actually taken from a external website carbon.now.sh to display code in prettier way. So it was advisable not to change stuff from it, hence i have put it as it is. Rest i have not used inline styling anywhere

**Review by Rishabh-078 — APPROVED** [review]
Code looks fine. You can merge it

**Review by manish-GH — APPROVED** [review]
Everything seems fine. It can be merged.

Next time do add the live app with the PR

**Pull request merged**